downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -147,8 +148,11 @@ abstract class VersionControlSystem {
         /**
          * Return glob patterns for files that should be checkout out in addition to explicit sparse checkout paths.
          */
-        internal fun getSparseCheckoutGlobPatterns(): List<String> =
-            LicenseFilenamePatterns.getInstance().allLicenseFilenames.generateCapitalizationVariants().map { "**/$it" }
+        internal fun getSparseCheckoutGlobPatterns(): List<String> {
+            val globPatterns = mutableListOf(ORT_REPO_CONFIG_FILENAME)
```

Comment:
For cases where multiple .ort.yml files are used we could use a pattern like `"*$ORT_REPO_CONFIG_FILENAME"`, then it would match all files as long as they follow the convention to end with `.ort.yml`.

Reply:
Good idea, will do.